product_selector.php

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
the name of this file is misleading.  Could it be changed to project-selector.php?  generally use - rather than _ for file names